OrbitCore/SamplingProfiler.cpp

```diff
@@ -74,12 +74,12 @@ void SamplingProfiler::AddCallStack(CallstackEvent& callstack_event) {
 }
 
 //-----------------------------------------------------------------------------
-void SamplingProfiler::AddHashedCallStack(CallstackEvent& a_CallStack) {
-  if (!HasCallStack(a_CallStack.callstack_hash())) {
+void SamplingProfiler::AddHashedCallStack(CallstackEvent& callstack_event) {
```

Comment:
Are both `AddCallstack` and `AddHashedCallstack` needed now? After recent changes they conceptually do the same thing, except that the former acts when `!HasCallStack` and the latter prints an error. Can this one just be removed or am I missing something?

Reply:
Removed AddHashedCallstack.